<launch>
 <rosparam command="load" file="$(find homer_navigation)/config/homer_navigation.yaml"/>
 <node ns="/homer_navigation" name="homer_navigation" pkg="homer_navigation" type="homer_navigation" output="screen" respawn="true"/> 
 <node name="drive_to_node" pkg="homer_navigation" type="drive_to" output="screen" respawn="true"/>
 <node name="detect_obstacle" pkg="homer_navigation" type="detect_obstacle.py" output="screen" respawn="true"/>
 <node name="depth_occupancy_map" pkg="homer_navigation" type="depth_occupancy_map" output="screen" respawn="true"/>
</launch>
